pkg/ibmvpc/vpc.go

```diff
@@ -319,6 +335,10 @@ func (nl *NaclLayer) Name() string {
 	return ""
 }
 
+func (nl *NaclLayer) ExtendedName(config *vpcmodel.VPCConfig) string {
+	return ""
+}
+
```

Comment:
```suggestion
```
seems this method is not needed?

Reply:
I think it is, see answer to the last comment